### PR TITLE
Add office365 active and inactive properties in reports graph api getoffice365servicesusercounts

### PR DESCRIPTION
**Pull request by luckyrass**
Add office365 active and office365 inactive properties in reports graph api getoffice365servicesusercounts

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[38a26d6](https://github.com/luckyrass/microsoft-graph-docs/pull/3757/commits/38a26d6e923722af56ffc4af75752ad405da6213)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 21 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[38a26d6](https://github.com/luckyrass/microsoft-graph-docs/pull/3757/commits/38a26d6e923722af56ffc4af75752ad405da6213)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[38a26d6](https://github.com/luckyrass/microsoft-graph-docs/pull/3757/commits/38a26d6e923722af56ffc4af75752ad405da6213)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/reportroot-getoffice365servicesusercounts.md](https://github.com/luckyrass/microsoft-graph-docs/blob/master/api-reference/beta/api/reportroot-getoffice365servicesusercounts.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/office365servicesusercounts.md](https://github.com/luckyrass/microsoft-graph-docs/blob/master/api-reference/beta/resources/office365servicesusercounts.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/luckyrass/microsoft-graph-docs/blob/master/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C11%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901111011266153-3757%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=z97J0kRKYrp7YMkkqy%2Blaoid%2BdovXN5hkonDYLnoza4%3D&st=2019-01-11T10%3A10%3A08Z&se=2019-02-11T10%3A15%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: concepts/changelog.md [hunk 1]
```diff
@@ -11,6 +11,14 @@ This changelog covers what's changed in Microsoft Graph, including the v1.0 and
 
 For details about known issues with Microsoft Graph APIs, see [Known issues](known-issues.md).
 
+## January 2019
+
+### Reports APIs
+
+| **Change type** | **Version** | **Description**                  |
+|:----------------|:------------|:-----------------------------------------|
+| Addition        | Beta version in WorldWide and Microsoft Graph China operated by 21Vianet | Added the **office365Active** and **office365Inactive** properties to the [office365ServicesUserCounts](/graph/api/resources/office365ServicesUserCounts?view=graph-rest-beta) entity.|
```
Comment (Lauragra) [inline concepts/changelog.md]: Regarding the version, to be clear, this update applies to the graph endpoint and the China cloud, but not the other national clouds? Ideally we would have a statement regarding Reports API implementation in national clouds in this topic: https://docs.microsoft.com/en-us/graph/deployments. We may also need a note calling this out in the API topics where we're adding these properties.

Comment (luckyrass) [reply concepts/changelog.md]: @Lauragra Thanks for your suggestion. We want to add those properties in **Microsoft Graph global service** and **Microsoft Graph China operated by 21Vianet**. How about change the value of **Version** to "Beta in Microsoft Graph global service and China operated by 21Vianet"? Besides, I don't know how to add a note in the API topics and why we need to do this. To my knowledge, all national clouds share the same graph API document.

File: api-reference/beta/resources/office365servicesusercounts.md [hunk 2]
```diff
@@ -22,7 +22,9 @@ localization_priority: Normal
 | yammerActive             | Int64  | The number of active users on Yammer. Any user who can post, read, or like messages is considered an active user. |
 | yammerInactive           | Int64  | The number of inactive users on Yammer.  |
 | teamsActive              | Int64  | The number of active users on Teams. Any user who posted messages in team channels, sent messages in private chat sessions, or participated in meetings or calls is considered an active user. |
-| teamsInactive            | Int64  | The number of active users on Teams.     |
+| teamsInactive            | Int64  | The number of inactive users on Teams.     |
+| office365Active          | Int64  | The number of active users on Office365.   |
+| office365Inactive        | Int64  | The number of inactive users on Office365.     |
```
Comment (Lauragra) [inline api-reference/beta/resources/office365servicesusercounts.md]: Please update the product names in the description column to "Microsoft Teams" and "Office 365" (with a space).

File: concepts/changelog.md (hunk 1 above)
Comment (Lauragra) [reply concepts/changelog.md]: If you call out the global service and the China national cloud, it seems that you're excluding the other national clouds. Is this update available in all the national cloud environments? This needs to be clarified and should be documented in the API topics and/or the deployments topic I linked to earlier. If it's available in all national clouds, then we just say that the version is "Beta" per the standard for this topic. Adding @dkershaw10  and @negoe for input.

File: api-reference/beta/resources/office365servicesusercounts.md (hunk 2 above)
Comment (luckyrass) [reply api-reference/beta/resources/office365servicesusercounts.md]: Nice Catch. I'll change it.

File: concepts/changelog.md (hunk 1 above)
Comment (luckyrass) [reply concepts/changelog.md]: Reports API are supported in Microsoft Graph global service and China operated by 21Vianet and not supported in other national clouds. So this update available in Microsoft Graph global service and China operated by 21Vianet. In this case. Should I add Reports API in this table?
image

Comment (Lauragra) [reply concepts/changelog.md]: Yes, please include an update to that table with your PR, and then in the Changelog we can just list "Beta" for the version. Thanks.

Comment (luckyrass) [reply concepts/changelog.md]: Thanks for your suggestion. I'll change it.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[02abc6a](https://github.com/luckyrass/microsoft-graph-docs/pull/3757/commits/02abc6ac543d5409fe16d2a7beb63d5ee94a53b6)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 22 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[02abc6a](https://github.com/luckyrass/microsoft-graph-docs/pull/3757/commits/02abc6ac543d5409fe16d2a7beb63d5ee94a53b6)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 55 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[02abc6a](https://github.com/luckyrass/microsoft-graph-docs/pull/3757/commits/02abc6ac543d5409fe16d2a7beb63d5ee94a53b6)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/reportroot-getoffice365servicesusercounts.md](https://github.com/luckyrass/microsoft-graph-docs/blob/master/api-reference/beta/api/reportroot-getoffice365servicesusercounts.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/office365servicesusercounts.md](https://github.com/luckyrass/microsoft-graph-docs/blob/master/api-reference/beta/resources/office365servicesusercounts.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/luckyrass/microsoft-graph-docs/blob/master/concepts/changelog.md) | :white_check_mark:Succeeded | |
[concepts/deployments.md](https://github.com/luckyrass/microsoft-graph-docs/blob/master/concepts/deployments.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C25%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901251005225644-3757%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=KsrCsTd2Tlapx1nhPpjiyxV4H%2Fm2sLQvq79HcyZM2eU%3D&st=2019-01-25T10%3A04%3A03Z&se=2019-02-25T10%3A09%3A03Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[ef570d9](https://github.com/luckyrass/microsoft-graph-docs/pull/3757/commits/ef570d9f3a65b007a382060b128c3782fabf7699)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/reportroot-getoffice365servicesusercounts.md](https://github.com/luckyrass/microsoft-graph-docs/blob/master/api-reference/beta/api/reportroot-getoffice365servicesusercounts.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/office365servicesusercounts.md](https://github.com/luckyrass/microsoft-graph-docs/blob/master/api-reference/beta/resources/office365servicesusercounts.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/luckyrass/microsoft-graph-docs/blob/master/concepts/changelog.md) | :white_check_mark:Succeeded | |
[concepts/deployments.md](https://github.com/luckyrass/microsoft-graph-docs/blob/master/concepts/deployments.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C1%5C25%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201901251928366671-3757%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=NQ88VwC3Ojrp%2B%2BOEnrmFfO90Z5VXvuUDk1c0v58%2FHx4%3D&st=2019-01-25T19%3A27%3A11Z&se=2019-02-25T19%3A32%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**